python/vyos/ifconfig/ethernet.py

```diff
@@ -253,6 +253,19 @@ def set_ufo(self, state):
         """
         return self.set_interface('ufo', state)
 
+    def set_ring_buffer(self, b_type, b_size):
+        """
+        Example:
+        >>> from vyos.ifconfig import EthernetIf
+        >>> i = EthernetIf('eth0')
+        >>> i.set_ring_buffer('rx', '4096')
+        """
+        cmd = '/sbin/ethtool -G {0} {1} {2}'.format(self.config['ifname'], b_type, b_size)
```

Comment:
please use self._cmd()

Reply:
Does not possible to use `self._cmd()` in this case, we need to check an exit code.

Reply:
Got it thanks!